### DISCUSSION TITLE
Unable to get minting dapp public

### DISCUSSION THREAD

**Discussion by progressaded** [hashlips-lab/nft-erc721-collection#330] [General]
Hi everyone  , I hope everyone is doing well  :)

After a hard 6 months of work I have finally completed my NFT collection  , but now I am stuck with the minting dapp  . So I am following the hashlips Masterclass and everything is going very well   , untill its time to deploy the minting dapp public  . 

When i open it on a local host everything is working just perfectly  , but when i opload the build and the html files to github and I go to github / settings / pages and I click on the link of my website  , it just opens a blank page  . I was really hoping someone could help me out with this or maybe tell me what I am doing wrong and if I am missing a step  .

Also when i look at my settings in github it says that this consosts of 100% html files  , is this normal?

Some help would be so much appreciated  ! I am also asking in the hashlips discord but so far nobody has responded me so far .

Have a nice day everyone 

![1](https://user-images.githubusercontent.com/107586541/174019953-0884b000-e027-4d10-9b08-7b24b235a7cb.png)
![2](https://user-images.githubusercontent.com/107586541/174019972-6d4bb8ac-18fe-481e-9259-49ff2339158b.png)
![3](https://user-images.githubusercontent.com/107586541/174019974-f3b8f212-d349-4ffc-a199-b7f912bc76b2.png)

**Comment by liarco** [comment]
Hi @progressaded,
please take a look at my reply here, I guess it's the same thing here: https://github.com/hashlips-lab/nft-erc721-collection/issues/324#issuecomment-1157346548

I hope this can be helpful. 😉

**Comment by progressaded** [comment]
Thank you sooooo much  , right no I am at work but when I get home I will check and I will comment here if its working or not  . Thank you so much for your time and thank you so much for all the effords you have put into all these so far  !

Have a lovely day :)

**Comment by progressaded** [comment]
Hi @liarco  , used Vercel and it worked  , I dont know how exacly because I just connected my Github to vercel and voila :) Thank you for your help and I hope you have a great weekend and a nice day !

**Reply by liarco** [reply]
That's great to hear! :)

As a reference for other users, we also have a [Video Tutorials](https://github.com/hashlips-lab/nft-erc721-collection#youtube-tutorials) section in our README which includes a [deployment tutorial for CloudFlare](https://youtu.be/uUrbIXUgVz4) which is a very powerful service (easy as much as Vercel if not even more). The configuration between the two should be almost the same.

**Reply by progressaded** [reply]
@liarco thank you so much for your fast responses  and all the hard work you have put into all this :)
